src/pages/dataElements/form/hooks.ts

```diff
@@ -0,0 +1,45 @@
+import { useDataQuery } from '@dhis2/app-runtime'
+import { useMemo } from 'react'
+import { Attribute } from '../../../types/generated'
+
+const CUSTOM_ATTRIBUTES_QUERY = {
+    attributes: {
+        resource: 'attributes',
+        params: {
+            fields: [
+                'id',
+                'mandatory',
+                'displayFormName',
+                'valueType',
+                'optionSet[*, options[id,displayName,name,code]]',
```

Comment:
Do we actually want `*` here? I think it would be better to explicitly list the filter, or use `:owner`?

Reply:
Already removed